pyproject.toml

```diff
@@ -16,5 +16,5 @@
 # under the License.
 [tool.black]
 line-length = 110
-target-version = ['py36', 'py37', 'py38']
+target-version = ['py36', 'py37', 'py38', 'py39']
```

Comment:
Do we also want to add 3.10, or do you only want to keep it in line with what is currently supported in Airflow?

Reply:
No yet. This is Airflow pyproject, not Breeze's one and currently py3.10 is not supported for airflow officially duet to mssql libraries not yet available (shoudl be there in January). But good eyes :)

Reply:
Totally missed the hierarchy - makes sense. Glad it's on the horizon! 🐍